mk/external.mk

```diff
@@ -34,14 +34,24 @@ EXTERNAL_SRC = LINUX BUSYBOX
 $(foreach T,$(EXTERNAL_SRC),$(eval $(download-n-extract)))
 
 # Build Linux kernel image
-LINUX_IMG = $(OUT)/bzImage
+ifeq ($(ARCH), x86_64)
+LINUX_IMG_NAME = bzImage
+LINUX_ARCH = x86
+else ifeq ($(ARCH), aarch64)
+LINUX_IMG_NAME = Image
+LINUX_ARCH = arm64
+else
+    $(error Unsupported architecture)
+endif
+LINUX_IMG = $(OUT)/$(LINUX_IMG_NAME)
```

Comment:
Rewrite as following:
```
LINUX_IMG := $(addprefix $(OUT)/,$(LINUX_IMG))
```

It is a matter of variable overriding. See https://www.gnu.org/software/make/manual/html_node/Overriding.html

Reply:
I pushed a new revision and changed `LINUX_IMG = $(OUT)/$(LINUX_IMG_NAME)` to `LINUX_IMG := $(addprefix $(OUT)/,$(LINUX_IMG_NAME))`

I kept the variable `LINUX_IMG_NAME` because it is used when compiling the kernel and copying the output file.